products/microos/main.pm

```diff
@@ -112,4 +121,8 @@ else {
     }
 }
 
+if (get_var 'SELINUX_BY_DEFAULT') {
```

Comment:
This is going to test SELinux after testing everything else..I don't mind that given we're meant to be testing the SELinux config as provided by default - if targeted breaks stuff we're testing earlier we'll learn about it..

the current code includes a test that is designed to change the SELinux config (see other comment). If I accept this as submitted the (incorrect) change to enforcing mode would actually be a no-op as it will immediately end the test run after doing so :)

for future reference, any tests you want to run before the other tests but after installation need to be done before lines that mention `load_feature_tests`

eg https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/products/microos/main.pm#L89 (for cases where the system is using the installer) and https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/products/microos/main.pm#L102 (for cases where booting from images)